config/webpack.config.prod.js

```diff
@@ -389,51 +97,15 @@ export default (() => {
           removeRedundantAttributes: true,
         },
       }),
-
-      // This gives some necessary context to module not found errors, such as the requesting resource.
-      new ModuleNotFoundPlugin(paths.appPath),
-
-      // Makes some environment variables available to the JS code, for example:
-      // if (process.env.NODE_ENV === 'development') { ... }. See `env.js`.
-      new webpack.DefinePlugin(env),
-
-      // Embed the small webpack runtime script in index.html
-      new InlineChunkHtmlPlugin(HtmlWebpackPlugin, [/runtime-.+[.]js/]),
```

Comment:
Does this not work in WP5 or with the version of `HtmlWebpackPlugin`?  It is useful not to need another download...

Reply:
The plugin [declares](https://github.com/facebook/create-react-app/tree/main/packages/react-dev-utils#new-inlinechunkhtmlpluginhtmlwebpackplugin-htmlwebpackplugin-tests-regex) it works with HtmlWebpackPlugin 4.x . Since we migrated to 5.x I've assumed it does not work.
I think also this usage conflicted with our cache config i.e.:
```
     // Keep the runtime chunk separated to enable long term caching
      runtimeChunk: 'single',
```